common/scripts/build-sct.sh

```diff
@@ -69,6 +67,24 @@ if ! [[ $BUILD_PLAT = IR ]] && ! [[ $BUILD_PLAT = ES ]] ; then
     exit
 fi
 
+. $TOP_DIR/../../common/config/common_config.cfg
```

Comment:
Please refactor the changes in file to a new file common/scripts/build-sct-arm.sh

The original file common/scripts/build-sct.sh shall be unmodified.